.github/workflows/nipoppy_track_processing_for_dataset.yaml

```diff
@@ -0,0 +1,120 @@
+name: Generate processing status files for OpenNeuroDatasets
+
+on:
+  workflow_dispatch:
+    inputs:
+      chunk_size:
+        description: "Number of datasets to process in each job"
+        required: false
+        default: "100"
```

Comment:
for now we only launch this manually?

Reply:
yes, until we implement a process to check SHAs of derivative repos!